### ISSUE TITLE
gradle 2.9 

### ISSUE THREAD

**Issue by qiujiayu** [ksoichiro/gradle-eclipse-aar-plugin#11]
use gradle 2.9, 
class org.gradle.mvn3.org.apache.maven.artifact.versioning.ComparableVersion not found.

**Comment by paulvi** [comment]
Do you mean something has changed within gradle 2.9 that this plugin does not work？

**Comment by paulvi** [comment]
Do you mean something has change within gradle 2.9 that this plugin does not work？

**Comment by paulvi** [comment]
Do you mean something has change within gradle 2.9 that this plugin does not work？

**Comment by qiujiayu** [comment]
Yes

**Comment by ksoichiro** [comment]
Thanks for reporting the issue. I'll check it soon.

**Comment by ksoichiro** [comment]
I confirmed the issue and fixed it.
Fixed version can be available as a snapshot.
Please update your build.gradle like this:

``` gradle
buildscript {
    repositories {
        mavenCentral()
        maven {
            url uri('https://oss.sonatype.org/content/repositories/snapshots/')
        }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:1.0.0'
        classpath 'com.github.ksoichiro:gradle-eclipse-aar-plugin:0.1.4-SNAPSHOT'
    }
}

apply plugin: 'com.github.ksoichiro.eclipse.aar'
```

I'll release it as v0.1.4 later.

**Comment by ksoichiro** [comment]
I've released the plugin v0.1.4, so I close this issue.
It will be synced to Maven Central in a couple of hours.

From this version, you can also use this plugin with plugin mechanism introduced in Gradle 2.1.

``` gradle
plugins {
  id "com.github.ksoichiro.eclipse.aar" version "0.1.4"
}
```

See README for detail.

**Comment by qiujiayu** [comment]
Thanks
